packages/workflows/src/definition/product/update-products.ts

```diff
@@ -350,6 +356,43 @@ const handlers = new Map([
       ),
     },
   ],
+  [
+    UpdateProductVariantsActions.upsertPrices,
+    {
+      invoke: pipe(
+        {
+          merge: true,
+          invoke: [
+            {
+              from: InputAlias.ProductsInputData,
+              alias: ProductHandlers.updateProducts.aliases.products,
+            },
+            {
+              from: UpdateProductsActions.prepare,
+            },
+          ],
+        },
+        ProductHandlers.upsertVariantPrices
+      ),
+      compensate: pipe(
+        {
+          merge: true,
+          invoke: [
+            {
+              from: UpdateProductVariantsActions.prepare,
+            },
+            {
+              from: UpdateProductVariantsActions.upsertPrices,
+              alias:
+                ProductHandlers.upsertVariantPrices.aliases
+                  .productVariantsPrices,
```

Comment:
question: since we are merging inputs, having an alias doesn't affect the input right? 
if this is the case we should remove it